rosbridge_server/src/rosbridge_server/websocket_handler.py

```diff
@@ -181,8 +180,8 @@ def on_message(self, message: str | bytes) -> None:
 
     @log_exceptions
     def on_close(self) -> None:
-        assert self.node_handle is not None
         cls = self.__class__
+        assert cls.node_handle is not None
```

Comment:
I assume the node handle should be a specific class. If so, better to check the node handle is an instance of that class.